pkg/volume/util/fsquota/quota_linux.go

```diff
@@ -269,11 +269,15 @@ func clearQuotaOnDir(m mount.Interface, path string) error {
 // don't cache the result because nothing will clean it up.
 // However, do cache the device->applier map; the number of devices
 // is bounded.
-func SupportsQuotas(m mount.Interface, path string) (bool, error) {
+func SupportsQuotas(m mount.Interface, path string, userNamespaceDisabled bool) (bool, error) {
 	if !enabledQuotasForMonitoring() {
 		klog.V(3).Info("SupportsQuotas called, but quotas disabled")
 		return false, nil
 	}
+	if userNamespaceDisabled {
+		klog.V(3).Info("SupportQuotas called and LocalStorageCapacityIsolationFSQuotaMonitoring enabled, but userNamespaceDisabled")
```

Comment:
I wonder if v3 is the right level here, and also `userNamespaceDisabled` doesn't give a user much feedback. Maybe "but pod is not confined by a user namespace, meaning it can change the fs quota, nullifying the feature" or something